clusterloader2/framework/framework.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/perf-tests/clusterloader2/api"
+	"k8s.io/perf-tests/clusterloader2/framework/util"
+)
+
+const (
+	automanagedNamespaceName = "Namespace"
+)
+
+func convertObjectTypeToGVK(objectType *api.ObjectType) schema.GroupVersionKind {
+	return schema.GroupVersionKind{
+		Group:   objectType.APIGroup,
+		Version: objectType.APIVersion,
+		Kind:    objectType.Kind,
+	}
+}
+
+type Framework struct {
+	frameworkClient *framework.Framework
```

Comment:
loading framework means loading half of the world...
Can we avoid that?

IIUC, the only usecase for that as of now is the "CreateNamespace" method. 
If so, maybe we can start with implementing this ourselves (we don't need majority of it at the beginning and leave a TODO to consider reusing that in the future if we end-up vendoring whole framework anyway?)

@shyamjvs for thoughts too

Reply:
I'm removing all references to "kubernetes...framework" (at least from framework).